client/landing/gutenboarding/components/domain-picker/style.scss

```diff
@@ -62,3 +62,11 @@
 .domain-picker__divider {
 	margin-top: 20px;
 }
+
+
+.domain-picker__price {
+	color: var( --studio-gray-20 );
+	size: 0.8em;
```

Comment:
Was this meant to be `font-size`? Can probably be removed if it already looks correct.